content/pages/meetup/2024/0904-nycu.rst

```diff
@@ -0,0 +1,127 @@
+========================================
+Meetup 2024 September 4th in NYCU
+========================================
+
+:date: 2024-08-27 15:58
+:url: meetup/2024/0904-nycu
+:save_as: meetup/2024/0904-nycu.html
+
+Date
+-----
+
+* Date: September 4th, 2024
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
+There will be many different subjects discussed at the same time.
+
++-----------------+--------------------------+-------------------------------+------------------------------------+
+| Timetable       | Topic 1                  | Topic 2                       | Topic 3                            |
++=================+==========================+===============================+====================================+
+| 18:30 ~ 19:30   | Free Chat                                                                                     |
++-----------------+--------------------------+-------------------------------+------------------------------------+
+| 19:30 ~ 21:00   | Free Chat                | Modmesh Basic Training        | Modmesh Development Discussion     |
++-----------------+--------------------------+-------------------------------+------------------------------------+
+| 21:00 ~ 21:30   | Free Chat                                                                                     |
++-----------------+--------------------------+-------------------------------+------------------------------------+
```

Comment:
timetable

Reply:
@visitorckw will not attend the meetup next week.

Reply:
<img width="916" alt="image" src="https://github.com/user-attachments/assets/f96da8ec-a87b-4fa3-955b-bd29fc459d8c">

It looks pretty nice!

@visitorckw, could you please let the team know when you plan to host a topic in the meetup?